tests/workflow/workflow_with_models.py

```diff
@@ -76,3 +78,41 @@
 dict_from_json = json.loads(demo_workflow_json)
 deserialized_demo_workflow = wf.Workflow.dict_to_object(dict_from_json)
 assert demo_workflow == deserialized_demo_workflow
+
```

Comment:
Maybe we should split this and set the test of to_script in a separate test file.
You can still import a workflow from dessia_common/models/workflows